Emitron/Emitron/UI/Library/LibraryView.swift

```diff
@@ -104,12 +114,17 @@ struct LibraryView: View {
         HStack {
           Image("sort")
             .foregroundColor(.textButtonText)
-
-          Text(filters.sortFilter.name)
-            .font(.uiLabelBold)
-            .foregroundColor(.textButtonText)
+          if isSorting {
```

Comment:
Hi @sammyd,  yes - you are correct. I didn't need to provide an isSorting property. I originally thought I needed it so I can force a UI update. I removed the property and added your code as well as some additional code for the disabled check. It works exactly as before. 👍

I didn't add a property to the repository because I'm thinking I don't think I'll need to change the repo state in such a way. I was thinking I would add it if I needed to do so in a different part of the codebase. Do you think that's a good idea or would you prefer I add it to the repository? 

Thanks!

Reply:
I don't think that it's necessary to add a property anywhere unless you want to. The only thing I was kinda alluding to is the repetition of the "check the state" code that appears twice. I think it's fine as it is :+1: